third_party/move/move-compiler-v2/tests/testsuite.rs

```diff
@@ -155,6 +155,29 @@ impl TestConfig {
                 generate_file_format: false,
                 dump_annotated_targets: verbose,
             }
+        } else if path.contains("/explicit-drop/") {
+            pipeline.add_processor(Box::new(LiveVarAnalysisProcessor {}));
+            pipeline.add_processor(Box::new(ReferenceSafetyProcessor {}));
+            pipeline.add_processor(Box::new(ExplicitDrop {}));
+            Self {
+                type_check_only: false,
+                dump_ast: false,
```

Comment:
Yes, phantom types are simply ignored. And there there is no check for phantom types in struct/function definitions. Since we need to land this PR quickly, I am planning to do this in a separate PR.

Reply:
Could we please add an issue as requested?